smartsim/settings/batchSettings.py

```diff
@@ -59,17 +120,12 @@ def __init__(
 
     @property
     def scheduler(self) -> str:
-        """Return the launcher name."""
-        return self._batch_scheduler.value
-
-    @property
-    def batch_scheduler(self) -> str:
-        """Return the scheduler name."""
+        """Return the scheduler type."""
         return self._batch_scheduler.value
 
     @property
     def scheduler_args(self) -> BatchArguments:
-        """Return the batch argument translator."""
+        """Return the BatchArguments child class."""
         return self._arguments
 
     @property
```

Comment:
For the ``env_vars`` property, should we be return a reference?  I think standard we were working for is that items set are deep copied and items returned are by reference.

Reply:
oop yes!